deploy/manifests/dev/us-east-2/cluster/storetheindex/flux-cd.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: source.toolkit.fluxcd.io/v1beta2
+kind: GitRepository
+metadata:
+  name: storetheindex
+spec:
+  interval: 5m
+  url: https://github.com/filecoin-project/storetheindex.git
+  ref:
+    # TODO: update to main branch once PR is ready for review.
+    branch: masih/infra-dev-setup
+---
+apiVersion: kustomize.toolkit.fluxcd.io/v1beta2
+kind: Kustomization
+metadata:
+  name: storetheindex
+spec:
+  serviceAccountName: flux
+  interval: 5m
+  path: "./deploy/manifests/dev/us-east-2/tenant/storetheindex"
+  sourceRef:
+    kind: GitRepository
+    name: storetheindex
+  wait: true
+  timeout: 2m
+  prune: false # TODO: Re-enable this once the team is used to GitOps.
+  images:
+    - name: storetheindex
+      newName: 407967248065.dkr.ecr.us-east-2.amazonaws.com/storetheindex/storetheindex
```

Comment:
This lets us say "storetheindex" as the container image but have that replaced by the ecr repo?

Reply:
That's right. That ECR is created but is empty. follow up PRs will set up build and push to this ECR along with flux image reconciliation policy to automatically deploy when images are pushed etc.